tests/Http/Three/SignupTest.php

```diff
@@ -9,6 +9,78 @@
 class SignupTest extends BrowserKitTestCase
 {
     /**
+     * Test that a POST request to /signups creates a new signup.
+     *
+     * POST /api/v3/signups
+     * @return void
+     */
+    public function testCreatingASignup()
```

Comment:
should I also add tests to make sure you can't do any of these things unauthorized? We have one for the delete endpoint but should each endpoint have a test for this?

Reply:
I think that's a good idea, just to be safe! 🔒

Reply:
ok cool! Do we want to have `GET /signups` and `GET /signups/:signup_id` be authorized requests as well? Right now they are not.

Reply:
I think we probably want that for the index... maybe not for the `/signups/:id` one?

Reply:
wait actually they all should be (https://github.com/DoSomething/rogue/blob/master/app/Http/Controllers/Three/SignupsController.php#L30) - but for some reason I am able to without this in Postman for these and also the PATCH method 🤔

Reply:
We [somewhat awkwardly](https://github.com/DoSomething/rogue/blob/1fe5bd4319dd3edcef73d53bb5b3a8016acac41c/app/Http/Middleware/AuthenticateApi.php#L23-L26) bypass this for all `GET` endpoints in the middleware, which we should probably remove in the future.

Reply:
ahhh! Is it ok to add `UPDATE` there for now?

Reply:
added the `UPDATE` for now but didn't for `GET` endpoints for now! @DFurnes